internal/core/command/main.go

```diff
@@ -78,3 +81,21 @@ func Main(ctx context.Context, cancel context.CancelFunc, router *mux.Router) {
 
 	// code here!
 }
+
+// MessageBusBootstrapHandler sets up the MessageBus connection if MessageBus required is true.
+// This is required for backwards compatability with older versions of 2.x configuration
+// TODO: Remove in EdgeX 3.0
+func MessageBusBootstrapHandler(ctx context.Context, wg *sync.WaitGroup, startupTimer startup.Timer, dic *di.Container) bool {
+	configuration := container.ConfigurationFrom(dic.Get)
+	if configuration.MessageQueue.Required {
+		if !handlers.MessagingBootstrapHandler(ctx, wg, startupTimer, dic) {
+			return false
+		}
+		if !handlers.NewExternalMQTT(nil).BootstrapHandler(ctx, wg, startupTimer, dic) {
```

Comment:
I assume the OnConnectHandler will be added later, correct?

Reply:
Yes I plan to implement it in #4075 or #4078